JS-Algorithms/challenges.js

```diff
@@ -95,7 +113,24 @@ Since 10! === 3628800 and you sum 3 + 6 + 2 + 8 + 8 + 0 + 0
 ***** */
 
 const digitSum = (n) => {
-  // YOUR CODE HERE...
+  if (!(n > 1)) return;
+  
+  const fact = (n) => {
```

Comment:
Better naming for variables and functions:

- The `fact` can be `calculateFactorial`
- Yo already had a variable named `n` it can be `calculatedNumber`

Remember that it's better to be very explicit with the names.